src/components/Transactions/Transaction.tsx

```diff
@@ -21,7 +21,6 @@ export interface transactionProps {
   amount: string;
   id: string;
   detailsButton: {
-    text: string;
     action: (id: string) => void;
   };
```

Comment:
I think it's pretty unuseless having an object (detailsButton) with just one property (action) after the text property removal. It's a good idea to have directly 'action' instead of 'detailsButton'